tachiyomiorg/tachiyomi-extensions
# MGKomik: Fix Latest, CF workaround

- using latest instead of new manga for latest
- add header to not be blocked by cloudflare
- add ratelimit
- change url request a little to bypass cloudflare check

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
